hw2/Views/MatchView.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  MatchView.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class MatchView: BaseView {
+    
+    private let timeLabel = UILabel()
+    private let timeStatusLabel = UILabel()
+    private let dividerImageView = UIImageView()
+    private let homeTeamLogoImageView = UIImageView()
+    private let awayTeamLogoImageView = UIImageView()
+    private let homeTeamLabel = UILabel()
+    private let awayTeamLabel = UILabel()
+    private let homeScoreLabel = UILabel()
+    private let awayScoreLabel = UILabel()
+    
+    override func addViews() {
+        super.addViews()
+        addSubview(timeLabel)
+        addSubview(timeStatusLabel)
+        
+        addSubview(dividerImageView)
+        
+        addSubview(homeTeamLogoImageView)
+        addSubview(homeTeamLabel)
+        addSubview(homeScoreLabel)
+        
+        addSubview(awayTeamLogoImageView)
+        addSubview(awayTeamLabel)
+        addSubview(awayScoreLabel)
+    }
+    
+    override func styleViews() {
+        super.styleViews()
+        
+        StylingHelper.styleImageView(imageView: dividerImageView, imageName: "DividerHorizontal")
```

Comment:
Za DividerHorizontal ti ne treba slika, to nek bude običan view width 1 sam odgovarajućim background colorom.